shadowsocks-csharp/Encryption/AEAD/AEADMbedTLSEncryptor.cs

```diff
@@ -80,19 +80,19 @@ public override void cipherEncrypt(byte[] plaintext, uint plen, byte[] ciphertex
                 case CIPHER_AES:
                     ret = MbedTLS.cipher_auth_encrypt(_encryptCtx,
                         /* nonce */
-                        _encNonce, (uint) nonceLen,

```

Comment:
we don't have an official code format yet, IMO we should keep their format to reduce diff.

Reply:
reverted code format as it was before